client/devdocs/design/test/gutenberg.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Gutenberg from '../gutenberg';
```

Comment:
we'll probably want to differentiate this name here from the editor itself since `Gutenberg` is the name of the whole project. we could make `../gutenberg-blocks` for example which I think would help clear up the confusion

Reply:
yep, it makes sense. I have just renamed everything to `gutenberg-blocks`